src/Test.Utilities/TestAdditionalDocument.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Test.Utilities
+{
+    /// <summary>
+    /// An implementation of AdditionalText meant specifically for unit-testing.
+    /// </summary>
+    public class TestAdditionalDocument : AdditionalText
```

Comment:
📝 This is not needed with the new test library

Reply:
Do you have an example/documentation?

Reply:
https://github.com/dotnet/roslyn-analyzers/blob/dba3183193df36d666fc48c0eac0d2d640d0041d/src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs#L105-L113

Reply:
+1

Reply:
@mavasani I'm not sure who is currently working on this, but I was expecting this to get removed

Reply:
Removed